src/apps/path-finder/store/search-thunk.ts

```diff
@@ -50,12 +50,12 @@ export function searchPath(
       cells.forEach((cell) => {
         grid[cell.row][cell.col] = cellType;
         visitedCellCount++;
+        dispatch(setVisitedCellCount(visitedCellCount));
```

Comment:
instead of incrementing it inside loop, we can just add the cells count

```js
visitedCellCount += cells.length;
dispatch(setVisitedCellCount(visitedCellCount));
```